float32 pos_x
float32 pos_y
float32 pos_z
float32 rot_r
float32 rot_p
float32 rot_y
float32 gripper

